frontend/components/ui/navbar.tsx

```
@@ -0,0 +1,225 @@
+                    <div className='flex items-center gap-4 lg:hidden'>
+                        <Link
+                            href='/login'
+                            className='bg-[#f4f4f4] text-[#6D6C80] hover:bg-[#5751E1] hover:text-white p-2 rounded-full'
+                        >
+                            <UserRound />
+                        </Link>
+
+                        {/* Mobile menu */}
+                        <Sheet>
+                            <SheetTrigger asChild>
+                                <button className='text-[#5751E1]'>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)